tests/unit/test_pathutil.py

```diff
@@ -491,3 +493,23 @@ def test_parse_rm_dirs__bad(dirs: List[str], err_msg: str):
     with pytest.raises(UserInputError) as exc:
         parse_rm_dirs(dirs)
     assert err_msg in str(exc.value)
+
+
+@pytest.mark.parametrize(
+    'expect, files, runN',
+    [
+        param(1, [], False, id='1st run (from filenames)'),
+        param(2, ['run1'], False, id='2nd run (from filenames)'),
+        param(1000, ['run999'], False, id='1000th run (from filenames)'),
```

Comment:
These two are the same test

Reply:
Yes, and that's a mistake. Will fix.

Reply:
Also realized that I should include test case: There is a file not in the form runX in the directory.